docs/articles/templates/notes-warnings/requesthook-onresponse-no-stub-headers.md

```diff
@@ -0,0 +1,3 @@
+> Important! You can not change response body or its headers inside a `RequestHook.onResponse()` call.
```

Comment:
You can change headers inside of request hooks.

`> Important! You can not change response body or its status code inside a `RequestHook.onResponse()` call.`

Reply:
```suggestion
> Important! You cannot change the response body or its status code inside a `RequestHook.onResponse()` call.
```